client/post-editor/editor-revisions/index.jsx

```diff
@@ -90,7 +94,7 @@ export default flow(
 		state => {
 			const postId = getEditorPostId( state );
 			const siteId = getSelectedSiteId( state );
-
+			const isRequestingRevisions = isRequestingPostRevisions( state, siteId, postId );
```

Comment:
Since `EditorRevisionsList` is already receiving `postId` and `siteId`, and since `isRequestingRevisions` is only used in that component, I think it'd be better to just include it there, instead of adding a layer.

Reply:
Good call, thanks for reviewing @michaeldcain <3